### ISSUE TITLE
[SEP1] stellar.toml has human-readable info but no i18n

### ISSUE THREAD

**Issue by msfeldstein** [stellar/stellar-protocol#464]
There are some human readable fields in the stellar.toml, for example CURRENCIES.name and CURRENCIES.desc. These fields show up in User Interfaces, but can only be set for one language, usually english.

There are two options here: 
1. Localize stellar.toml.  This will solve the problem, but requires moving from what used to be a static file to something that requires a good amount of infrastructure to support.  This is a pretty large barrier, especially since this is often hosted on a domain supported by a different part of the organization, where its much easier to just drop a static file than to run arbitrary code.  You also have to maintain a second localization pipeline.
2. Move human readable things from the toml file to the API layer.  Most of the fields in question are in the CURRENCIES area, used by SEP6/24.  SEP6/24 has the `/info` endpoint which is localized already and is intended to provide information to the apps to display to users before starting a deposit.

There are also some human readable fields for VALIDATORS and ORG_INFO which seem much lower priority since this is looked up ad hoc and not really used in any UI anywhere except for maybe developers exploring the network overlay

**Comment by msfeldstein** [comment]
Solution 1 is referenced in [one of the SEP26 proposals](https://github.com/stellar/stellar-protocol/pull/460) while [the other](https://github.com/stellar/stellar-protocol/pull/454) will need solution 2

**Comment by efritze** [comment]
There's a third option: localized stellar.toml files.

e.g. 

* stellar.toml (default)

* stellar_es.toml (Spanish localizations)

* stellar_fr.toml (French localizations)

**Comment by leighmcculloch** [comment]
I think option 2 solves the problem while creating the least number of new problems.

An anchor is going to need to provide localization on their API endpoints already and to localize the toml file would duplicate that effort.

It keeps the toml file focused on providing information to machines rather than mixing its audience.

To localize the toml file would also mean duplicating most of the data in it across multiple files, unless a hierarchy of files is designed. If there's a hierarchy a client will need to make multiple requests if these are just static files. The complexity climbs quickly I think. 

The only human readable attribute that I see that can't be served by the anchor API endpoint is the organization description. We could allow only that one field to have multiple languages and make it explicit in the SEP that generally fields requiring localization don't belong in the toml file.

My thoughts are that it's not worth complicating the toml file for one field, beyond supporting localization of only the org description.

**Comment by efritze** [comment]
We have to consider assets that are not anchored. i.e. pure Stellar assets (like SLT, MOBI, RMT), if we are to use /info for i18n rather than localizing stellar.toml.

These assets don't have a transfer server, hence nowhere to localize their asset metadata.

If we're going the /info route, I propose the /info endpoint be made independent of any transfer protocol. Maybe a new SEP focused on /info with support for new fields like contact: {email, twitter,...}  (an organization may have different contact information for each of their assets, ... e.g. btc_support@anchor.com, eur_support@anchor.com).

**Comment by msfeldstein** [comment]
At first thought this feels right to me (pulling /info out into it's own thing).  It can also solve a problem we'll see where the /info endpoint clashes between different transfer protocols.

**Comment by tempofr** [comment]
we had info multilingual and have had descriptions in multiple languages.

I still am not sure you want to stuff so much data into the toml file. We added this 3 years ago with just an underscore and the lang. Individual toml files works. Maybe add a country extension as well.

desc="XOF is a CFA currency use for cash in and cash out in west africa"
desc_fr="XOF est une monnaie (CFA) utilisée encaisser et encaisser en Afrique de l'ouest"
countries="BJ CI SN TG"

**Comment by efritze** [comment]
Check out the SEP27 #474 proposal for asset localization.

We simply took the concepts of /info, extended it, and made it independent of any transfer protocol.

No need to localize inside stellar.toml.
